src/TurbulenceClosures/turbulence_closure_implementations/scalar_diffusivity.jl

```diff
@@ -121,7 +121,7 @@ function ScalarDiffusivity(time_discretization=ExplicitTimeDiscretization(),
     κ = convert_diffusivity(FT, κ; discrete_form, loc, parameters)
     ν = convert_diffusivity(FT, ν; discrete_form, loc, parameters)
 
-    return ScalarDiffusivity{typeof(time_discretization), typeof(formulation), required_halo_size}(ν, κ)
+    return ScalarDiffusivity{typeof(time_discretization), typeof(formulation), typeof(required_halo_size)}(ν, κ)
```

Comment:
This is wrong, the last type parameter is supposed to be an integer like 1, 2, 3, 4